create-topics.sh

```diff
@@ -28,14 +28,11 @@ fi
 
 # Expected format:
 #   name:partitions:replicas:cleanup.policy:<key>=<value>:<key>=<value>...
-IFS=','; for topicToCreate in $KAFKA_CREATE_TOPICS; do
+IFS=';'; for topicToCreate in $KAFKA_CREATE_TOPICS; do
```

Comment:
I think that was just changed in another PR today and so you should wait till it's approved.

Reply:
OK, I will wait and rebase